opencti-platform/opencti-front/src/utils/Colors.test.ts

```diff
@@ -0,0 +1,12 @@
+import { describe, it, expect } from 'vitest';
+import { hexToRGB } from './Colors';
+
+describe('Function: hexToRGB', () => {
+  it('should return matching rgb color', () => {
+    expect(hexToRGB('#70D907', 1)).toEqual('rgb(112, 217, 7, 1)');
+  });
+
+  it('should return default alpha value', () => {
+    expect(hexToRGB('#70D907')).toEqual('rgb(112, 217, 7, 0.1)');
```

Comment:
10% alpha by default ? wut

Reply:
I don't know, I just write tests on an existing function to illustrate the documentation I am writing but I didn't look more in how this function is used